test/extended/security/scc.go

```diff
@@ -270,7 +270,10 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
 		pod, err := exutil.NewPodExecutor(oc, "restrictedcapsh", image.ShellImage())
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		// TODO: remove desiredCapabilities once restricted-v2 is the default
+		// system:authenticated SCC in the cluster - in favour of alternativeDesiredCapabilities
 		desiredCapabilities := "000000000000051b"
```

Comment:
note to myself (also maybe worth a comment?): this corresponds to `cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_setpcap,cap_net_bind_service`

Reply:
yes please :-)

Reply:
these caps are going to be dropped once the related PR merges, I don't see much value describing it during its final days as it has lived undescribed for what I'm going to guess is more than 5 years